.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,24 @@
+# .github/workflows/chromatic.yml
```

Comment:
Have you tested this workflow in your forked repo? If yes, kindly share the workflow run link.

Reply:
Yes it's working

Reply:
Can you provide the link when you executed in your forked repo?

Reply:
https://github.com/devilkiller-ag/asyncapi-website/actions/runs/9373475111/job/25807339881